integration/templates/ingress_controller_values.go

```diff
@@ -6,4 +6,10 @@ package templates
 // node ports.
 const IngressControllerValues = `ingressController:
   legacy: true
+
+controller:
+  resources:
+    requests:
+      cpu: 500m
+      memory: 600Mi
```

Comment:
e2e tests environment seems to be constrained so without adjusting resource usage like this, 3rd replica wasn't able to start, due to not enough of resources.

Reply:
We could maybe increase the size of the kind nodes but I think this makes more sense. :+1: